com.unity.netcode.gameobjects/Runtime/Core/NetworkBehaviour.cs

```diff
@@ -576,13 +576,17 @@ private void NetworkVariableUpdate(ulong clientId, int behaviourIndex)
                                 m_NetworkVariableIndexesToReset.Add(k);
                             }
 
+                            var bytesReported = NetworkManager.LocalClientId == clientId
+                                ? 0
```

Comment:
nit: a comment explaining why we return 0 here might be useful (like "not a failure but success with 0") — also applies to other 0 returning ternaries below.